cosmos/views.py

```diff
@@ -1,11 +1,31 @@
 # from django.http import HttpResponse
+import datetime
+import random
+
 from django.http import HttpResponse
 from django.shortcuts import render
 from django.template import loader
 
+from apps.users.models import Profile
+from cosmos.constants import FOUNDING_DATE
+
 
 def index(request):
-    return render(request, "index.html")
+    members = Profile.objects.count()
+    nationalities = Profile.objects.values("nationality").distinct().count()
+    active_years = int((datetime.date.today() - FOUNDING_DATE).days // 365.25)
+    events_amount = random.randint(0, 69)  # TODO include actual event number
```

Comment:
Just keep it a static number for now, having it be a random number makes no sense, especially as that means it would change on reload.